timeless/customers/models.py

```diff
@@ -44,7 +44,7 @@ def create_by_poster(cls, poster_customer: dict):
         Method should return Customer object with given data from
         poster_customer dict
         """
-        return Customer(
+        return cls(
```

Comment:
@vladarefiev I would leave it as ``Customer`` here, it's clearer than ``cls``, I believe.

Reply:
@amihaiemil I didn't agree, but if you want I will do it

Reply:
@vladarefiev Is there any difference in functionality? I guess not. If it's just for reading purposes, then let it be ``Customer`` pls :D

Reply:
@amihaiemil done